p2p/p2p.go

```diff
@@ -68,7 +69,7 @@ type Config struct {
 }
 
 // NewService creates a new p2p.Service using the service config. It initializes the host and dht
-func NewService(conf *Config, msgChan chan<- Message) (*Service, error) {
```

Comment:
We don't want to change this. You can read from the initial channel you make in the tests, you just can use `sb.msgChan`